src/jit/compiler.h

```diff
@@ -1082,8 +1082,9 @@ class LclVarDsc
 #ifdef FEATURE_HFA
         assert(lvIsHfa());
         return HfaTypeFromElemKind(_lvHfaElemKind);
-#endif // FEATURE_HFA
+#else
```

Comment:
```
# else // !FEATURE_HFA
#endif // !FEATURE_HFA
```

Reply:
In general, when the `#ifdef`s are small non-nested like this, we don't need to comment them:
```
Do specify the macro name in a comment at the end of the closing #endif of a long or nested #if/#ifdef.
```
https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#75-commenting-ifdefs
and personally I find that it makes it quite cluttered.

That said, if there *is* a closing comment, there should similarly be a comment on the `#else` (i.e. `!FEATURE_HFA` in this case), and the `#endif` should have the "!"

Reply:
I also don't think that for one line ifdefs we need topcomment the #else